sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -877,7 +877,7 @@ def setup(self, data_sampler=None):
 
       # See fn_data in dataflow_runner.py
       fn, args, kwargs, tags_and_types, window_fn = (
-          pickler.loads(self.spec.serialized_fn))
+        pickler.loads(self.spec.serialized_fn))
```

Comment:
Wasn't the prior indentation correct?

Reply:
A yapf run likely changed it, I'll double-check